analyses/snv-callers/compare_snv_callers_plots.Rmd

```diff
@@ -327,6 +328,42 @@ GGally::ggpairs(vaf_df %>% dplyr::select(-VAF_consensus),
 ggplot2::ggsave(file.path(plots_dir, "vaf_correlations_plot.png"))
 ```
 
+## Mutation Region barplot
+
+Summarize `Variant_Classification` information per caller. 
+
+```{r}
+# Summarize by the number of times each base change shows up in each category.
+perc_var_df <- long_vaf_df %>% 
+  # Summarize the number of mutations per caller
+  dplyr::count(caller, Variant_Classification, name = "count") %>%
+  dplyr::add_count(caller, wt = count) %>%
+  # Calculate the percent of each 
+  dplyr::mutate(percent = count / n)
+```
+
+Make a barplot illustrating the percent of the mutations for each caller that 
+that are each type of change. 
+
+```{r}
+perc_var_df %>%
+  ggplot2::ggplot(ggplot2::aes(x = Variant_Classification, y = percent)) +
+  ggplot2::theme_classic() +
+  ggplot2::geom_bar(
+    position = "dodge", stat = "identity",
```

Comment:
Trying to decide if these will look better as stacked plots. Your call

Reply:
How would they be stacked?

Reply:
You should just be able to drop the `position="dodge"`

Reply:
Ah, so same variables just stacked.